performance/src/main/java/org/hibernate/validator/performance/BenchmarkRunner.java

```diff
@@ -29,11 +26,12 @@
  */
 public final class BenchmarkRunner {
 
-	private static final List<Class<?>> DEFAULT_TEST_CLASSES = Arrays.asList(
-			SimpleValidation.class,
-			CascadedValidation.class,
-			StatisticalValidation.class
-	);
+	private static final Stream<? extends Class<?>> DEFAULT_TEST_CLASSES = Stream.of(
+			"org.hibernate.validator.performance.simple.SimpleValidation",
+			"org.hibernate.validator.performance.cascaded.CascadedValidation",
+			"org.hibernate.validator.performance.statistical.StatisticalValidation",
+			"org.hibernate.validator.performance.multilevel.MultiLevelContainerValidation"
```

Comment:
Maybe add a comment just before starting the BV 2 classes:
`// Benchmarks specific to Bean Validation 2.0`
It would be even more important to explain why they are not added in the same way as the others.